### DISCUSSION TITLE
'Push-to' no Nexus eFinder version

### DISCUSSION THREAD

**Discussion by AstroKeith** [WimDeMeester/eFinder#58] [Ideas]
Another variant!
A UK user is adding eFinder to his push to dobsonian which doesnt have encoders!

Its early days, but we have a new beta variant that ...

Uses a gps dongle instead of the Nexus to get location and date time
On demand does a capture and plate solve.
Generates a feed to SkySafari (via a standalone router)

We're going to experiment with

An accelerometer to see if the scope has moved.
This will hopefully allow the eFinder to do continual capture/solves.
The logic being,.....
if the scope has moved but stopped, start a capture/solve
if it moves during capture, dump the image and try again.
(Solving a bad image just hangs up the flow)

I might also start looking at Tetra3 as the solver. Its apparently fast, but tricky!

I'll report on progress, but any ideas welcome!

**Comment by AstroKeith** [comment]
Forgot to add....

I've started to setup ramdisks on my builds and save the working images and files to that. Doesnt speed up the solve but SD cards can wear out quite quickly.
I've also stopped saving copies of images to /Stills.

**Reply by WimDeMeester** [reply]
@mrosseel This might also be interesting for you!

**Comment by mrosseel** [comment]
I think the ramdisk idea was already implemented in the pr. The rest sounds
indeed exactly like the pifinder, it's a logical extension of efinder
